### ISSUE TITLE
ErrorInvalidPtx, Unable to compile .cu and run .ptx file

### ISSUE THREAD

**Issue by Nucs** [jdermody/brightwire#24]
**Tl;dr:** I've tried to compile and run but cuda's JIT won't accept my compiled .ptx file.

My old GPU did not support 3.0 compute so I've purchased a new `GIGABYTE GeForce GTX 1060 3GB MINI ITX OC` which has 6.1 compute-compatibility.
I've ensured I have the required vc++ and in-fact nvcc compiles the .cu file without any error, not even in verbose mode.

This is the command I've used to compile:
`nvcc brightwire.cu -use_fast_math -ptx -m 64 -arch compute_61 -code sm_61 -o brightwire.ptx`

The exception I've received is:
```
ManagedCuda.CudaException
  HResult=0x80131500
  Message=ErrorInvalidPtx: This indicates that a PTX JIT compilation failed.
  Source=ManagedCuda
  StackTrace:
   at ManagedCuda.CudaContext.LoadModule(String modulePath)
```

I've tried:
* With arch and code compute_60, 
* With `-code sm_61`
* Uninstalled `Cuda 10.0` (after unsuccessful tries) and installed `Cuda 9.2`.
* Without `-use_fast_math`
* Ensured the GPU is indeed the one I mentioned above (via CudaContext, after it is initialized in CudaProvider).
* Ensured the `.ptx` that fails loading is indeed the one I compiled.
* Ensured via `CudaContext` that my GPU indeed supports 6.1 gpu architecture.
More information:
### `nvcc -V`
```
nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2018 NVIDIA Corporation
Built on Tue_Jun_12_23:08:12_Central_Daylight_Time_2018
Cuda compilation tools, release 9.2, V9.2.148
```

### Verbose output during compilation.
https://pastebin.com/JXr5Cz8t

**Comment by jdermody** [comment]
I haven't seen that particular error before, but I DID find some problems when I used cuda 10 for the first time as the driver that came with the CUDA 10 toolkit didn't work with my graphics card.

I suggest:

1. uninstalling all versions of all cuda libraries
2. reinstalling cuda 10
3. install the exact driver for your graphics card from the nvidia website (make sure the cuda 10 driver is replaced)
